pkg/api/types.go

```diff
@@ -1844,3 +1846,43 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+// SecurityContext holds security configuration that will be applied to a container.  SecurityContext
+// contains duplication of some existing fields from the Container resource.  These duplicate fields
+// will be populated based on the Container configuration if they are not set.  Defining them on
+// both the Container AND the SecurityContext will result in an error.
+type SecurityContext struct {
+	// Capabilities are the capabilities to add/drop when running the container
+	// DUPLICATE OF CONTAINER FIELD
```

Comment:
replace `DUPLICATE OF CONTAINER FIELD` with `Must match Container.Privileged or be unset.`